test-cases/tests/installation/a40-rhoam-on-osd-trial.md

```diff
@@ -18,42 +16,31 @@ Verify RHOAM installation on OSD Trial works as expected.
 
 ## Steps
 
-1. Provision an OSD Trial Cluster
+1. Provision an OSD Trial Cluster through Jenkins using the `managed-api-install-master` pipeline
```

Comment:
I would add a link to the pipeline here. Syntax:  `[text of the link](url)`

Reply:
lgtm otherwise

Reply:
Should be good now